packages/core/src/cli/commands.ts

```diff
@@ -58,6 +72,11 @@ export function runCli(): void {
 
   inspectCommand
     .description('inspect the Rsbuild / Rspack configs of Rslib projects')
+    .option(
+      '--lib <name>',
```

Comment:
shared options can be set in `CommonOptions`

Reply:
There's a bit different in the text. 

```txt
'build the specified library (may be repeated)',
'inspect the specified library (may be repeated)',
```

Reply:
use `choose the specified library` is enough, and what is `may be repeated` mean?